cmd/export.go

```diff
@@ -49,6 +49,10 @@ To export everything about a dataset, use the --dataset flag.`,
 		requireNotRPC(cmd.Name())
 		path := cmd.Flag("output").Value.String()
 		format := cmd.Flag("format").Value.String()
+		dataFormat := cmd.Flag("data-format").Value.String()
+		if dataFormat != "" && !(dataFormat == "json" || dataFormat == "csv" || dataFormat == "cbor") {
```

Comment:
In the future we should think about adding a function to the dataset package that makes this a litle less verbose, but that doesn't exist so this is great :)

Reply:
This comment makes me happy, because my instinct was to do just that but I didn't follow it. So next time I'll follow it.